.travis.yml

```diff
@@ -59,6 +59,8 @@ matrix:
     env: CONDA_ENV=py36-bottleneck-dev
   - python: 3.6
     env: CONDA_ENV=py36-condaforge-rc
+  - python: 3.6
+    env: CONDA_ENV=py36-pynio-dev
```

Comment:
add this same block to just after line 41. Travis requires these `allowed_failures` tests to be listed in the main test matrix too.

Reply:
Gotcha. Done.